### DISCUSSION TITLE
sending email in the form

### DISCUSSION THREAD

**Discussion by rcarvalho2clicks** [Budibase/budibase#11206] [Help] [answered]
Hello I have a form and when the person clicks save it needs to send an email how do I do that?

**Comment by tglynx** [comment]
Hi,

in short: 
Define a new action on the 'Save' Button -> Trigger Automation
Create a new automation from there -> Send-EMail

Let me know if this helps already, or if you have further questions.

Regards,
Michael

**Comment by rcarvalho2clicks** [comment]
Michael got it right and now how do I get the contents of the table and
put it in the body of the email?

Em qua., 12 de jul. de 2023 às 06:59, Michael Sommer <
***@***.***> escreveu:

> Hi,
>
> in short:
> Define a new action on the 'Save' Button -> Trigger Automation
> Create a new automation from there -> Send-EMail
>
> Let me know if this helps already, or if you have further questions.
>
> Regards,
> Michael
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/11206#discussioncomment-6425553>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAG55XQFZXDJ67XH2XDCPZLXPZYO7ANCNFSM6AAAAAA2GQNKUY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
<http://www.jpiaget.com.br> *Rodrigo Damasceno de Carvalho*
Supervisor de Tecnologia da Informação

(11) 4367-8303 / (11) 99124-9874
***@***.***

**Comment by tglynx** [comment, marked as answer]
Hi,

again in short: 
Go to your automation (in automate) and have a look at the 'Trigger' section.
Add the necessary fields for the content you want to pass.

Could look like this as an example: 
![Bildschirm­foto 2023-07-12 um 17 42 23](https://github.com/Budibase/budibase/assets/30650435/7132d31e-c62b-42fc-91fb-4c00924df7f5)

In the SendEmail Step of the Automation use the defined fields in the HTML Contents.
Could look like this as an example: 
![Bildschirm­foto 2023-07-12 um 17 43 46](https://github.com/Budibase/budibase/assets/30650435/2738c6a3-a679-4f1a-bdbe-6499e396d61d)

Back in design edit the actions of your save button again and add the fields in the 'Trigger Automation' action.
Pass the necessary information using 'Bindings'.

Could look like this (as an example).
![Bildschirm­foto 2023-07-12 um 17 40 30](https://github.com/Budibase/budibase/assets/30650435/249704c5-dbba-4c08-9ea0-60dd81d8eec2)

The necessary bindings depend on what you want to pass. 

Hope this helps,
Michael
